src/app/cases/page.tsx

```diff
@@ -203,20 +204,35 @@ export default function Page() {
           <ListingCount $color={COLORS.greyMid}>
             {filteredCases.length} listings found
           </ListingCount>
-          {filteredCases.map(c => (
-            <ListingCard
-              key={c.id}
-              caseData={c}
-              isSelected={c.id === selectedCard}
-              onClick={() => {
-                setSelectedCard(c.id);
-                setCaseInfo(c);
-              }}
-            />
-          ))}
+          {filteredCases.length === 0 ? (
+            <CenteredH3 $color={COLORS.greyMid}>No cases listed</CenteredH3>
+          ) : (
+            <>
+              {filteredCases.map(c => (
+                <ListingCard
+                  key={c.id}
+                  caseData={c}
+                  isSelected={c.id === selectedCard}
+                  onClick={() => {
+                    setSelectedCard(c.id);
+                    setCaseInfo(c);
+                  }}
+                />
+              ))}
+            </>
+          )}
         </CardColumn>
         <CaseDetailsContainer>
-          {caseInfo && <CaseDetails caseData={caseInfo} />}
+          {caseInfo ? (
+            <CaseDetails caseData={caseInfo} />
```

Comment:
not here, but in the `useEffect`, can we also set selectedCard to the UUID of the first card? to make it obvious that the first card is selected by default

Reply:
we can also make a state that tracks whether or not cases have been loaded, then set that to true in the `.then` handler when we load the data (also in `useEffect`). this lets us handle the loading cases state (could show nothing for now, or leave for later)